examples/quilljs/index.js

```diff
@@ -1,69 +1,153 @@
 if(Quill == null){
-  alert("You must download quilljs! It needs to be in the same directory as y-richtext!");
+    alert("You must download quilljs! It needs to be in the same directory as y-richtext!");
```

Comment:
Lets agree on two spaces, ok? We also use two spaces in coffeescript. I recently started to work on the Yjs wiki. There I wrote a short article about [code conventions](https://github.com/y-js/yjs/wiki/Code-Conventions)